frontend/app/src/components/StreamlitDialog/StreamlitDialog.test.tsx

```diff
@@ -19,8 +19,9 @@ import React, { Fragment } from "react"
 import { screen } from "@testing-library/react"
 
 import { mockSessionInfo, render, SessionInfo } from "@streamlit/lib"
+import { DialogType } from "@streamlit/app/src/components/StreamlitDialog/constants"
 
-import { DialogType, StreamlitDialog } from "./StreamlitDialog"
+import { StreamlitDialog } from "./StreamlitDialog"
```

Comment:
nit: If there is no technical reason, I would keep the import style the same for the imports here, so either relative or absolute